### PR TITLE
feat: Implement video tab on/off switch similar to wizard switches and show pop-up

### LINKED ISSUE #6416: Virtual Events: Implement video tab on/off switch similar to wizard switches and show pop-up
The video event feature should be available for up to 25 people free of charge. Some online events might not want the video feature and they have the option to add details elsewhere. Organizers should therefore be able to switch the feature on and off. 

Before we implement details of the invoice process, we need to give organizers the option to actually do switch the feature on and off and provide information about the functionality and costs to users. 

Therefore please implement the following
* Add a switch button to switch the feature on and off similar to the wizard e.g. the attendee form
* When the user switches the feature on, show a pop up with the following text 
   * "The video room feature is available free of charge for events with up to 25 tickets. For events with more than 25 ticket registrations a one-time charge of 2 USD (or 2 Euro for Euro events) will be added per ticket. This fee will be charged monthly for any new registrations only. It is non-refundable in case ticket holders cancel their registration or if the feature is switched off at a later time. By activating this feature you agree to pay the one-time fee per ticket. Do you want to activate this feature now and agree to pay the additional ticket fee per ticket if the number of your tickets is above 25 tickets?"
   * Yes -> "Yes - activate video rooms now. I agree to pay the fees." / No -> "No, do not activate it."
* When an organizer switches off the feature show this 
    * "If you switch off the feature, video rooms will no longer be available for ticket holders. Data you entered will no longer be visible for your attendees and speakers. Fees for ticket holders you might have paid already will not be refunded. Do you really want to switch off this video feature?
    * "Switch off."-> switches it off. "Continue using video rooms." -> leaves them on as is.
* Do not delete any video room data when the user switches it off. Simply deactivate the UI.

Screenshot from 2021-01-18 17-11-46

Screenshot from 2021-01-18 13-49-08

### PR DESCRIPTION
**Pull request by ranjitkshah**
Fixes #6416 
reqd #https://github.com/fossasia/open-event-server/pull/8029

#### Short description of what this resolves:
https://www.loom.com/share/34446dfb79764177a6d5e028bf951007

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Review by iamdevvalecha — APPROVED** [review]
NICE! LGTM

**Comment by ranjitkshah** [comment]
We can access video rooms (even if feature is disabled) by going directly to something like `http://localhost:4200/e/b6ac502f?side_panel=true`
We need to hide video rooms from the side panel as well.
What if chat is also disabled then side panel would be empty, should it be disabled as well?
@mariobehling What do you suggest?

**Comment by maze-runnar** [comment]
just resolve conflict for now, and it's good to go. For above issue you mentioned, we will see later.

File: app/templates/events/view/videoroom/list.hbs [hunk 1]
```diff
@@ -4,48 +4,76 @@
...
+
 <Modals::AddRoomModal
   @event={{this.model.event}}
   @isOpen={{this.isRoomModalOpen}}
   @onAdded={{action this.refreshModel}} />
+
+<Modals::SimpleModal
+  @isOpen={{this.isFeatureModalOpen}}
+  @action={{this.toggleVideoRoom}}
+  @actionName={{if this.model.event.isVideoroomEnabled 'switches it off. "Continue using video rooms.' 'Yes - activate video rooms now. I agree to pay the fees.'}}
```
Comment (maze-runnar) [inline app/templates/events/view/videoroom/list.hbs:75]:
```suggestion
  @actionName={{if this.model.event.isVideoroomEnabled 'switches it off. Continue using video rooms.' 'Yes - activate video rooms now. I agree to pay the fees.'}}
```

**Review by maze-runnar — APPROVED** [review]

**Comment by mariobehling** [comment]
Please take out the pop-up as we are currently not able to add the payment feature for this part.

**Review by sachinchauhan2889 — CHANGES_REQUESTED** [review]
By default isVideoEnabled is false.
This means for existing events also isVideoEnabled is also false. This means we will not see join button on existing events. Organizer will need to go to video section to enable videoroom.

**Comment by ranjitkshah** [comment]
> By default isVideoEnabled is false.
> This means for existing events also isVideoEnabled is also false. This means we will not see join button on existing events. Organizer will need to go to video section to enable videoroom.

we will do that manually on prod db for all events. If I create new migrations it will reflect conflicts with previous migration.

**Review by sachinchauhan2889 — APPROVED** [review]
LGTM after manually migration.

**Review by MeghalBisht — APPROVED** [review]

**Pull request merged**